src/components/Button/Button.tsx

```diff
@@ -44,16 +47,26 @@ export const Button = ({
   const fullStyles = isFull ? 'w-full' : '';
   const centerStyles = isCenter ? 'relative left-1/2 -translate-x-1/2 ' : '';
   const disabledStyles = isDisabled ? styles.disabled : '';
+  const iconSize = size === 'lg' ? '6' : '4';
+  // const forDisabled = isDisabled ? styles['disabled'] : '';
+  const stylesType = isDisabled ? styles['disabled'] : styles[isType];
```

Comment:
I like this change... What do you think about just adding `disabled` as one of the types, then conditionally adding `disabled={isDisabled}` based on isType equalling disabled?

Reply:
I was thinking the same. made this change. this makes more sense